### PR TITLE
updated with latest faker documentation

### PR DESCRIPTION
**Pull request by captv89**
**Parent issue** 

The sample JSON had outdated faker functions. Replaced them with the latest ones.

### CODE REVIEW

File: samples/generate-mock-data.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "uuid": "",
   "lastMigration": 13,
-  "name": "Tutorial - Generate mock data",
+  "name": "Mock-Data",
```
Comment (255kb) [inline samples/generate-mock-data.json]: I would rather keep the original title as this mock is linked to the "Generate mock data" [tutorial](https://mockoon.com/tutorials/generate-mock-json-data/) of the website.

**Review by 255kb — CHANGES_REQUESTED** [review]
Thank you for the PR. It looks good. 
I would just prefer to keep the title if you can revert it. 
Thank you.

File: samples/generate-mock-data.json (hunk 1 above)
Comment (captv89) [reply samples/generate-mock-data.json]: Yes sure. Changes updated.

**Review by 255kb — APPROVED** [review]

**Comment by 255kb** [comment]
Thanks!

**Pull request merged**